onnxruntime/test/optimizer/graph_transform_test.cc

```diff
@@ -5157,6 +5157,74 @@ TEST_F(GraphTransformationTests, PropagateCastOpsTests_Gelu) {
 
 #endif
 
+#ifdef ENABLE_TRAINING_CORE
```

Comment:
you can merge this macro with pervious test case: 

"#endif

#ifdef ENABLE_TRAINING_CORE"